frontend/src/app/prices/page.jsx

```diff
@@ -0,0 +1,45 @@
+import Cards from "../../components/card/Card";
+export default function Pricing() {
+    return (
+        <div className="bg-[#EFEBF5] flex flex-col items-center  p-6 sm:p-8">
+        <h1 className="text-3xl sm:text-4xl font-medium bg-white text-gray-800 text-center p-4 border-2 max-w-screen-2xl w-full border-gray-300 rounded-t-lg"> عروض الأسعار</h1>
+        <div className="border-2 bg-[#EDE1F0] bg-opacity-32 rounded-b-lg shadow-lg p-20  sm:pt-20 max-w-screen-2xl w-full">
+          <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-4">
+            {Cards.map((plan, index) => (
+              <div key={index} className="bg-white rounded-3xl shadow-lg pb-6 text-center flex flex-col justify-between transform hover:scale-105 transition-transform duration-300 max-w-xl min-h-full" >
```

Comment:
Great work ! for more improvement , I suggest to convert this code into a re-usable (PricingCard.jsx) component